browser/ui/brave_ads/ad_notification_view.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_BRAVE_ADS_AD_NOTIFICATION_VIEW_H_
+#define BRAVE_BROWSER_UI_BRAVE_ADS_AD_NOTIFICATION_VIEW_H_
+
+#include <string>
+
+#include "base/callback_forward.h"
+#include "brave/browser/ui/brave_ads/ad_notification.h"
+#include "ui/views/animation/ink_drop_host_view.h"
+#include "ui/views/metadata/metadata_header_macros.h"
+
+namespace gfx {
+class Canvas;
+class Point;
+}  // namespace gfx
+
+namespace brave_ads {
+
+class AdNotificationView : public views::InkDropHostView {
```

Comment:
I'm not a views guru, so can't thoroughly review UI code, but please make sure that all views define correct accessibility properties (e.g. `GetAccessibleNodeData`). This is important, because the whole browser may crash (in certain circumstances) if they are wrong

Reply:
Added accessibility

Reply:
for future reference, it would be extremely helpful to have separate commits for new changes, to simplify reviewing